test/helpers/kubectl.go

```diff
@@ -2001,6 +2004,18 @@ func (kub *Kubectl) WaitPolicyDeleted(pod string, policyName string) error {
 	return WithTimeout(body, fmt.Sprintf("Policy %s was not deleted in time", policyName), &TimeoutConfig{Timeout: HelperTimeout})
 }
 
+// WaitPodDeleted waits for pods with the given name to be deleted.
+func (kub *Kubectl) WaitPodDeleted(pod string) error {
```

Comment:
This would be a bit more reusable if it took the namespace (here `DefaultNamespace`) as an argument.